Incorporate Bayesian measurement error procedure into estimateCellCounts

sandrews5 opened the pull request:
New functionality to estimateCellCounts that will allow for eosinophil estimation in cord blood using the adult blood reference panel. Can be extended for additional cell types in the future.

kasperdanielhansen commented:
I don't understand why there are no updates to the documentation besides a new reference. The code suggests that you're automatically doing Bayes estimation under a certain scenario. I would at a minimum expect documentation updates explaining what is happening, and I would also have expected that this could be forced-enabled/disabled by setting an option to `estimateCellCounts()`.

sandrews5 commented:
Thanks for the suggestions Kasper. I've added a new argument to `estimateCellCounts` so that the user can turn off this behavior if needed. And I've also added a short little paragraph to better explain when the function invokes the measurement error procedure.
